### PR TITLE
ccmlib/scylla_node: pass `scylla.yaml` to `scylla sstable`

### PR DESCRIPTION
**Pull request by fruch**
* since we want to use this for encrypted sstable, we need to pass in the scylla.yaml

* since some of the output has binary data inside it, we need to make the call using bytes and not text

### CODE REVIEW

File: ccmlib/scylla_node.py [hunk 1]
```diff
@@ -1404,6 +1404,14 @@ def run_scylla_sstable(self, command, additional_args=None, keyspace=None, dataf
...
         if additional_args is None:
             additional_args = []
+
+        additional_args += ['--scylla-yaml-file', str(Path(self.get_conf_dir()) / common.SCYLLA_CONF)]
+        if keyspace:
+            additional_args += ['--keyspace', keyspace]
+
+        if column_families and len(column_families) == 1:
+            additional_args += ['--table', column_families[0]]
+
```
Comment (tchaikov) [inline ccmlib/scylla_node.py:1414]: i'd suggest create yet another wrapper method around `run_scylla_sstable()` and pass `addition_args` to with the expected argument instead, like

```py
def dump_sstable_scylla_metadata(self, keyspace, column_family):
    # ...
    additional_args = ['--keyspace', keyspace,
                       '--table', column_family]
    scylla_metadata = self.run_scylla_sstable('dump-scylla-metadata',
                                               additional_args,
                                               keyspace=keyspace,
                                               column_families=[column_family])
```

Comment (fruch) [reply ccmlib/scylla_node.py:1414]: including the yaml file ?

Comment (fruch) [reply ccmlib/scylla_node.py]: I've moved it to a new function, to minimize the changes to the other functions

**Review by denesb — APPROVED** [review]

File: ccmlib/scylla_node.py [hunk 2]
```diff
@@ -1583,6 +1585,41 @@ def dump_sstable_stats(self,
         stdout, _ = sstable_stats['']
         return json.loads(stdout)['sstables']
 
+    def dump_sstable_scylla_metadata(self,
+                           keyspace: str,
+                           column_family: str,
+                           datafiles: Optional[List[str]] = None) -> List[Dict[str, Any]]:
+        """dump scylla metadate `scylla sstable dump-scylla-metadata`
```
Comment (tchaikov) [inline ccmlib/scylla_node.py:1592]: s/metadate/metadata/

Comment (fruch) [reply ccmlib/scylla_node.py]: fixed

**Pull request merged**